apps/nextjs-website/react-components/components/Header/components/MenuDropdown.tsx

```diff
@@ -0,0 +1,155 @@
+import {
+  Stack,
+  type StackProps,
+  Typography,
+  useTheme,
+  useMediaQuery,
+  Link,
+  type Theme,
+} from '@mui/material';
+import { useState } from 'react';
+import ArrowDropDownIcon from '@mui/icons-material/ArrowDropDown';
```

Comment:
```suggestion
import { ArrowDropDownIcon } from '@mui/icons-material';
```
Consistency with other cases